app/src/main/java/com/example/kuba10/mypokemonplaces/ChooseFragment/ChooseFragment.java

```diff
@@ -1,98 +0,0 @@
-package com.example.kuba10.mypokemonplaces.ChooseFragment;
-
-
-import android.content.Context;
-import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.support.v7.widget.GridLayoutManager;
-import android.support.v7.widget.RecyclerView;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-import android.widget.LinearLayout;
-
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.FragmentListener;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-
-import java.util.ArrayList;
-
-
-public class ChooseFragment extends Fragment {
-
-
-    private PokemonImageAdapter pokeAdapter;
-    private GridLayoutManager gridManager;
-    private RecyclerView recyclerView;
-    private LinearLayout errorImage;
-    private FragmentListener fragmentListener;
-    private ArrayList<PokemonGo> pokemonGo_data_list;
-
-
-    public static ChooseFragment newInstance(ArrayList<PokemonGo> pokemonGo_data_list) {
-        ChooseFragment fragment = new ChooseFragment();
-
-        Bundle args = new Bundle();
-        args.putParcelableArrayList(Constants.LIST, pokemonGo_data_list);
-        fragment.setArguments(args);
-        return fragment;
-
-
-    }
```

Comment:
Wystrzegaj się tak wielu pustych linijek. Rozumiem, że one trochę pomagają oddzielić kod ale wyraźnie przeszkadzają go w czytaniu innym i są zupełnie nieprofesjonalne.

Reply:
Jest jakiś automat który wywala puste linijki? Plugin, albo coś?

Reply:
Nie kojarzę. Moim automatem dopóki tak nie przestałem robić był kolega stojący nad głową :smile:

Reply:
File -> Settings -> Editor -> Code Style -> Java(or Scala or whatever your language is) -> Blank Lines

Teraz ctrl - alt - L usuwa puste linie - koniec zmartwien